### PR TITLE
eagerly load Field object type

### PR DESCRIPTION
**Pull request by Marco-Daniel**
The union fields on type FormField (like TextBoxField, SubmitField etc) break without this line.

### CODE REVIEW

**Comment by Marco-Daniel** [comment]
Added some code to make the field label position an enum. Just not sure what composer command to run to add it to the autoloading process

File: src/type/enum/class-field-label-pos-enums.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
```
Comment (AND101190) [inline src/type/enum/class-field-label-pos-enums.php:1]: http://192.168.1.18/wordpress/graphql

**Review by AND101190 — COMMENTED** [review]
katherine.acosta@hotmail.es

File: src/class-type-registry.php [hunk 2]
```diff
@@ -32,6 +33,7 @@ class Type_Registry {
 	public function init( TypeRegistry $type_registry ) {
 		// enums.
 		Id_Type_Enums::register();
+		Field_Label_Pos_Enums::register();
```
Comment (AND101190) [inline src/class-type-registry.php:36]: 676200104

File: src/type/object/class-field-type.php [hunk 3]
```diff
@@ -40,9 +40,10 @@ public static function register() {
 			register_graphql_object_type(
 				$type_name,
 				[
-					'description' => $field->get_nicename(),
```
Comment (AND101190) [inline src/type/object/class-field-type.php:43]: Im_kthe

**Pull request merged**